apps/nrm/views/partials/check-your-answers-so-far-summary-table-potential-victim.html

```diff
@@ -0,0 +1,252 @@
+<h2 class="details-header">{{#t}}pages.confirm.sections.potential-victim.header{{/t}}</h2>
+<table id="potential-victim" class="table-details">
+  <tbody>
+      {{^isDutyToNotifyPath}}
+      {{#values.does-pv-need-support}}
+        <tr>
+          <td>{{#t}}pages.confirm.fields.does-pv-need-support.label{{/t}}</td>
+          <td class="no-white-space">
+            <span>{{formattedDoesPvNeedSupport}}</span>
+          </td>
+          <td></td>
+        </tr>
+      {{/values.does-pv-need-support}}
+      {{/isDutyToNotifyPath}}
+
+      {{#values.pv-want-to-submit-nrm}}
+        <tr>
+          <td>{{#t}}pages.confirm.fields.pv-want-to-submit-nrm.label{{/t}}</td>
+          <td class="no-white-space">
+            <span>{{formattedPvWnatToSubmitNrm}}</span>
```

Comment:
Wnat?

Reply:
lol cheers think I need to edit that one throughout @cmedlandho  could create a JIRA issue for this when you have some time. Think we will need to edit all occurrences of that.

Reply:
ahh no worries